pkg/providers/vsphere/config/template.yaml

```diff
@@ -170,20 +170,45 @@ spec:
       apiServer:
         extraArgs:
           cloud-provider: external
-{{- if .extraArgs }}
-{{ .extraArgs.ToYaml | indent 10 }}
+          anonymous-auth: "false"
+          audit-policy-file: /etc/kubernetes/audit-policy.yaml
+          audit-log-path: /etc/kubernetes/audit.log
+          audit-log-maxage: "30"
+          audit-log-maxbackup: "10"
+          audit-log-maxsize: "512"
+          profiling: "false"
+          request-timeout: "1m0s"
+{{- if .apiserverExtraArgs }}
+{{ .apiserverExtraArgs.ToYaml | indent 10 }}
 {{- end }}
+        extraVolumes:
+        - hostPath: /etc/kubernetes/audit-policy.yaml
+          mountPath: /etc/kubernetes/audit-policy.yaml
+          name: audit-policy
+          pathType: File
+          readOnly: true
+        - hostPath: /etc/kubernetes/audit.log
+          mountPath: /etc/kubernetes/audit.log
+          name: audit-log
+          pathType: FileOrCreate
+          readOnly: false
       controllerManager:
         extraArgs:
           cloud-provider: external
+          profiling: "false"
+          terminated-pod-gc-threshold: "100"
```

Comment:
is 100 the recommendation here, it seems a bit low?

Reply:
I used the recommendation, but I see the default is 12500 or something